chirp/drivers/kguv980p.py

```diff
@@ -646,6 +647,19 @@ def _oem_str_encode_1000GPLUS(in_str):
     return out_str
 
 
+def _str_decode_950(in_str):
+    out_str = ''
+    for c in in_str:
+        if c > 95:
+            out_str += ''
+        elif c >= 80:
+            out_str += chr(c-48)
+        else:
+            out_str += chr(c+48)
```

Comment:
Can you explain what you're doing here? It kinda looks like maybe you're just needing to mask out `0x80` from each of these characters, but I'm not sure what you're trying to accomplish here. Is the character set in the radio just starting at zero (for the character `0`)? Maybe `0x80` is just a flag for "capitals" or something? Either way, doing this with magic decimal numbers doesn't make this very understandable by other people with no comments.